### DISCUSSION TITLE
What is the Yellow on Measure name mean in TOM Explorer?

### DISCUSSION THREAD

**Discussion by fausttiger007** [TabularEditor/TabularEditor3#632] [🙋♀️TE3 Q&A] [answered]
What does the Yellow indicator on Measure name mean in TOM Explorer?

There are measures = another measure divided or multipled by a constant.  (convert mins to hours, vice versa, etc.)

![image](https://user-images.githubusercontent.com/905135/185270482-69f592a8-8c9a-4e76-ba09-db6732ab8344.png)

Tony

**Comment by fausttiger007** [comment, marked as answer]
It appears to indicate the measure value multiplied by constant is qualified
measure = table[measure2] * 60   (yellow indicator)
measure = measure2 * 60  (no yellow indicator)

**Reply by otykier** [reply]
Indeed - measure references should not be qualified by the table name. This produces a warning which is what the yellow dot indicates. Red dot would be syntax/semantic error.
